lib/datadog/core/telemetry/v1/product.rb

```diff
@@ -14,6 +14,13 @@ def initialize(appsec: nil, profiler: nil)
             @appsec = appsec
             @profiler = profiler
           end
+
+          def to_h
+            {
+              appsec: @appsec.to_h,
+              profiler: @profiler.to_h
+            }
+          end
```

Comment:
I haven't been following along, but is there any doc for this telemetry format? Why are appsec and profiler inside a "producs object", but tracing isn't?

Reply:
I got this from: https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v1/SchemaDocumentation/Schemas/products.md

They seem to only expect `appsec` and `profiler` at the moment

Reply:
Hmmm... I guess they built it to be more generic and handle languages where appsec/profiling is separate, although it seems a quite biased model -- tracer is definitely the preferred child >_>

I'll bring it up with the team, but yeah I guess let's live with this for now